brand/tests/test.py

```diff
@@ -217,3 +221,36 @@ def test_brand_invalid_tag(self):
 
         with self.assertRaises(ValidationError):
             brand1.save()
+
+
+class GetContactsAPITestCase(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.token = settings.REST_API_CONTACT_SINGLE_TOKEN
```

Comment:
It might be best to mock this out so that it's not dependent on the user having a good .env